lnbits/core/helpers.py

```diff
@@ -0,0 +1,42 @@
+import importlib
+import re
+
+from loguru import logger
+
+from lnbits.extension_manager import Extension
+
+from . import db as core_db
+from .crud import update_migration_version
+
+
+async def migrate_extension_database(ext: Extension, current_version):
+    try:
+        ext_migrations = importlib.import_module(f"{ext.module_name}.migrations")
+        ext_db = importlib.import_module(ext.module_name).db
+    except ImportError as e:
+        logger.error(e)
+        raise ImportError(
+            f"Please make sure that the extension `{ext.code}` has a migrations file."
+        )
+
+    async with ext_db.connect() as ext_conn:
+        await run_migration(ext_conn, ext_migrations, current_version)
+
+
+async def run_migration(db, migrations_module, current_version):
```

Comment:
is it possible to type the arguments? i can not really follow what is happening here in the next lines

Reply:
Done!
However, I've used `migrations_module: Any`. I could not figure out what the type for all modules is.
`typeof(migrations_module)` simply returns `module`